src/PolyhedralGeometry/Serialization.jl

```diff
@@ -1,170 +1,131 @@
+using JSON
+
 ##############################################################################
 """
-    save_cone(Cone, String)
+    save(Cone, String)
 
 Save a cone to a file in JSON format. The first argument is the cone, the
 second argument is the filename.
 """
-function save_cone(C::Cone, filename::String)
+function save(C::Cone, filename::String)
    bigobject = pm_object(C)
    Polymake.save_bigobject(bigobject, filename)
 end
 
-"""
-    load_cone(String)
-
-Load a cone stored in JSON format, given the filename as input.
-"""
-function load_cone(filename::String)
```

Comment:
An alternative approach would be to replace `load_cone(str)` by `load(Cone, str)` (where `Cone` literally is the type, passed in as an argument). I.e.
```
function load(::Type{Cone}, filename::String)
```

Reply:
Well, but then the user needs to provide what they think this object is anyway. I can do this though, if this is preferred.

Reply:
I can't say what is "preferred" as I still don't know what the big plan is for using these things. All I am saying is: if one is willing to write `load_cone(str)` then one should also be willing to accept `load(Cone, str)`, but that one has the advantage of being easier to use in generic code.